boa/src/builtins/error/type.rs

```diff
@@ -16,24 +16,50 @@
 //! [mdn]: https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/TypeError
 
 use crate::{
-    builtins::{function::make_builtin_fn, function::make_constructor_fn},
-    object::ObjectData,
+    builtins::BuiltIn,
+    object::{ConstructorBuilder, ObjectData},
+    property::Attribute,
     BoaProfiler, Context, Result, Value,
 };
 
 /// JavaScript `TypeError` implementation.
 #[derive(Debug, Clone, Copy)]
 pub(crate) struct TypeError;
 
-impl TypeError {
-    /// The name of the object.
-    pub(crate) const NAME: &'static str = "TypeError";
+impl BuiltIn for TypeError {
+    const NAME: &'static str = "TypeError";
+
+    fn attribute() -> Attribute {
+        Attribute::WRITABLE | Attribute::NON_ENUMERABLE | Attribute::CONFIGURABLE
+    }
+
+    fn init(context: &mut Context) -> (&'static str, Value, Attribute) {
+        let _timer = BoaProfiler::global().start_event(Self::NAME, "init");
 
+        let error_prototype = context.standard_objects().error_object().prototype();
+        let attribute = Attribute::WRITABLE | Attribute::NON_ENUMERABLE | Attribute::CONFIGURABLE;
+        let range_error_object = ConstructorBuilder::with_standard_object(
```

Comment:
`range_error_object` should be `type_error_object`, right?

Reply:
Copy paste mistake